addons/stock/models/stock_orderpoint.py

```diff
@@ -498,7 +498,7 @@ def _prepare_procurement_values(self, date=False, group=False):
             'group_id': group or self.group_id,
         }
 
-    def _procure_orderpoint_confirm(self, use_new_cursor=False, company_id=None, raise_user_error=True):
+    def _procure_orderpoint_confirm(self, use_new_cursor=False, company_id=None, raise_user_error=True, force_to_max=False):
```

Comment:
I would avoid an extra kwargs for this. Is it not possible to just set the qty_to_order in action_replenish and let _procure_orderpoint_confirm as it was before?

Reply:
Hello @amoyaux 
we can avoid an extra kwargs with context but setting `qty_to_order` in `action_replenish` I think it's not a good idea because we need to do a loop for this in `action_replenish` and every time we need to write `qty_to_order`. 

with the current code, we can avoid extra SQL queries.